### PR TITLE
Creates login controller

### PR DESCRIPTION
**Pull request by we3x**

### CODE REVIEW

File: app/scripts/auth/controller.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+'use strict'
+
+module.examples[
```
Comment (vuxor) [inline app/scripts/auth/controller.js:3]: should it be "module.exports"?

Comment (Crawleyprint) [inline app/scripts/auth/controller.js:3]: Yes, it's corrected in feature/auth branch.

File: app/scripts/auth/controller.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  '$scope',
+  'loginService',
+  function loginControler($scope, loginService) {
+    $scope.user = {
+      email: '',
```
Comment (vuxor) [inline app/scripts/auth/controller.js:8]: In markup I used username as value. For consistency we should stick to only one of them.

Comment (Crawleyprint) [inline app/scripts/auth/controller.js:8]: Yes, this should be changed in markup. REST API needs 'email' field in order to authenticate user on backend

**Pull request merged**